firewood/src/shale/mod.rs

```diff
@@ -139,12 +138,7 @@ impl<T: Storable> Obj<T> {
     }
 
     #[inline(always)]
-    pub fn get_space_id(&self) -> SpaceId {
-        self.value.get_mem_store().id()
-    }
-
-    #[inline(always)]
-    pub const fn from_typed_view(value: StoredView<T>) -> Self {
+    pub const fn from_stored_view(value: StoredView<T>) -> Self {
```

Comment:
renamed because there is no such thing as `typed_view` (I presume there used to be?)